opencti-platform/opencti-front/src/private/components/threats/threat_actors_individual/AddThreatActorIndividualDemographic.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FunctionComponent, useState } from 'react';
+import { ThreatActorIndividual_ThreatActorIndividual$data } from '@components/threats/threat_actors_individual/__generated__/ThreatActorIndividual_ThreatActorIndividual.graphql';
+import IconButton from '@mui/material/IconButton';
+import { Add } from '@mui/icons-material';
+import Drawer from '@components/common/drawer/Drawer';
+import CountryCreation from '@components/locations/countries/CountryCreation';
+import AddThreatActorIndividualDemographicLines, {
+  addIndividualsThreatActorIndividualLinesQuery,
+} from '@components/threats/threat_actors_individual/AddThreatActorIndividualDemographicLines';
+import { AddThreatActorIndividualDemographicLinesQuery } from '@components/threats/threat_actors_individual/__generated__/AddThreatActorIndividualDemographicLinesQuery.graphql';
+import { PreloadedQuery, usePreloadedQuery } from 'react-relay';
+import { useFormatter } from '../../../../components/i18n';
+import SearchInput from '../../../../components/SearchInput';
+import useQueryLoading from '../../../../utils/hooks/useQueryLoading';
+import Loader, { LoaderVariant } from '../../../../components/Loader';
+
+interface AddThreatActorIndividualDemographicComponentProps {
+  threatActorIndividual: ThreatActorIndividual_ThreatActorIndividual$data,
+  relType: string,
+  title:string,
+  queryRef: PreloadedQuery<AddThreatActorIndividualDemographicLinesQuery>,
+}
+
+const AddThreatActorIndividualDemographicComponent: FunctionComponent<
+AddThreatActorIndividualDemographicComponentProps
+> = ({
+  threatActorIndividual,
+  relType,
+  title,
+  queryRef,
+}) => {
+  const { t_i18n } = useFormatter();
+  const [open, setOpen] = useState<boolean>(false);
+  const [search, setSearch] = useState<string>('');
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const handleSearch = (term: string) => setSearch(term);
+
+  const data = usePreloadedQuery<AddThreatActorIndividualDemographicLinesQuery>(
+    addIndividualsThreatActorIndividualLinesQuery,
+    queryRef,
+  );
+
+  return (<div>
+    <IconButton
+      color='primary'
+      style={{ marginTop: '-11px' }}
+      onClick={handleOpen}
+    >
+      <Add fontSize="small" />
+    </IconButton>
+    <Drawer
+      open={open}
+      onClose={handleClose}
+      title={t_i18n(title)}
```

Comment:
provide an already translated title in the props to avoid t_i18n with a variable (that is not detected when we run extract-translation): we prefer t_i18n with string